kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiBaseResultSet.java

```diff
@@ -338,6 +338,10 @@ private Object evaluate(TTypeId columnType, Object value) {
         if (value instanceof String) {
           return ((String) value).getBytes();
         }
+      case INT_TYPE:
+        if (value instanceof String) {
+          return Integer.valueOf((String) value);
```

Comment:
why this is required?

Reply:
I think this should be handled on the JDBC engine side, convert the Oracle-specific metadata and values to the Hive style

Reply:
Well, I've tried it and implemented it on the JDBC engine side! Please review it~